src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -16,36 +16,55 @@ export const QuickLinks: HomeViewSection = {
 
 const QUICK_LINKS: Array<NavigationPill> = [
   {
-    title: "Follows",
-    href: "/favorites",
-    ownerType: OwnerType.follows,
-    icon: "FollowArtistIcon",
+    title: "Saves",
+    href: "/favorites/saves",
+    ownerType: OwnerType.saves,
+    icon: "HeartStrokeIcon",
   },
   {
     title: "Auctions",
     href: "/auctions",
     ownerType: OwnerType.auctions,
-    icon: "AuctionIcon",
+    icon: "GavelIcon",
   },
   {
-    title: "Saves",
-    href: "/favorites/saves",
-    ownerType: OwnerType.saves,
-    icon: "HeartIcon",
+    title: "New This Week",
+    href: "/collection/new-this-week",
+    ownerType: OwnerType.collection, // TODO: collection id?
```

Comment:
I think since this navigates to a specific collection we might need some additional tracking behavior now?